### PR TITLE
test undercloud nodes roles names

### PR DESCRIPTION
**Pull request by rrasouli**
Testing nodes role names
{{pytest: cfme/tests/infrastructure/test_roles_names.py --use-provider tripleo}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 2e902c140393b8f2219991350df9333ff3d7fbb5:

`cfme/tests/infrastructure/test_roles_names.py`:
- :bangbang: Line 5:1: **E302** *expected 2 blank lines, found 1*
- :red_circle: Line 7:9: **F821** *undefined name 'testgen'*
- :red_circle: Line 8:5: **F821** *undefined name 'testgen'*
- :bangbang: Line 16:1: **E303** *too many blank lines (3)*
- :red_circle: Line 16:2: **F821** *undefined name 'pytest'*
- :heavy_exclamation_mark: Line 28:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 578e5bedb1becd252814ab73c9a59586c5ad72ea:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4342b9fff7c28b7680a1d13f80efa0c68d31c79a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by jkrocil** [comment]
From IRC:
```
<jkrocil> rrasouli, you need to add a {{pytest: cfme/tests/infrastructure/test_roles_names.py --use-provider tripleo}} to your PR #3268
<jkrocil> as last line of your PR description
<jkrocil> because if you look at your PR in PRT, there were no tests run
```

**Comment by psav** [comment]
tripleo is available in our yamls? as key tripleo?

**Comment by rrasouli** [comment]
{{pytest: cfme/tests/infrastructure/test_roles_names.py --use-provider rhos8_uc}}

**Comment by psav** [comment]
Needs to be in the PR description

**Comment by mfalesni** [comment]
I initiated a retest since it was not executed with these parameters yet.

**Comment by mfalesni** [comment]
Now it only needs squashing. :+1:

**Comment by psav** [comment]
Please squash asap, then we can merge

**Comment by dajoRH** [comment]
Lint report for commit 2e7325594249df9cbc9ac58ca5af7f415129b025:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@rrasouli still not squashed.

**Comment by dajoRH** [comment]
Lint report for commit 8732c9acaf04b7dc67e11e75c07371d1bbce5e35:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/web_ui/__init__.py [hunk 1]
```diff
@@ -1498,6 +1499,8 @@ def _fill_form_list(form, values, action=None, action_always=False):
                 # TypeError - when loc is not resolvable to an element, elements() will yell
                 # vvv An alternate scenario when element is not resolvable, just wait a bit.
                 time.sleep(1)
+            except TimedOutError:
+                logger.warning("This element [{}] couldn't be waited for".format(loc))
```
Comment (psav) [inline cfme/web_ui/__init__.py]: These changes are already in the code base

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d2a189d041608b094d926e8de98812beb680f62b:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6a2bf21e454301bcc9080d7d180a58cb5efbed86:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f22eb7d9bb76f65f04abdd1593f22d71782f185e:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
So, please create a frech branch from fresh master and create a new PR from it.

Next time please watch out where are you rebasing to. Remember the maximum number you can put in the `git rebase -i HEAD~N` is the number of your commits on your branch. Otherwise it will go crazy.

**Pull request closed without merging**